forward43/setup_elasticsearch.py

```diff
@@ -0,0 +1,95 @@
+import json
+import os
+import pandas as pd
+
+import utils.elasticsearch as es_utils
+
+
+def create_index_with_mapping(es):
+
+    # FIXME: The mappings should be retrieved from the mappings PR
+    mappings = {
+        "properties": {
+            "name": {
+                "type": "keyword"
+            },
+            "creator": {
+                "type": "text"
+            },
+            "pledged": {
+                "type": "float"
+            },
+            "state": {
+                "type": "text"
+            }
+        }
+    }
+
+    is_created = es_utils.create_index(
+        es_object  = es, 
+        index_name = 'forward43', 
+        mappings   = mappings,
+        n_shards   = 1,
+        n_replicas = 0
+    )
+
+
+def read_json_data(es):
+
+    # FIXME: This assumes that data is stored in ../data folder. Only kickstarter_scraper does this.
+    #        1. Do the same for the other scrapers
+    #        2. Add a path argument
+    data_path_common = '../data/'
```

Comment:
This can get dangerous when you repeat it like in line 65. 
Patterns that have worked for me in the past is to add a `forward43/data/constants.py` script with the following code:
```
import os
from pathlib import Path

DATA_DIRECTORY = os.path.dirname(os.path.realpath(__file__))
```
This will allow you to do a 
```
from forward43.constants import DATA_DIRECTORY
```
Then on both the crawling, and the write to ES side you can refer to that constant when you need access to what is now `data_path_common` and it is portable in that when you refactor a folder to somewhere else, it'll automatically move the constant with it.